apps/ledger-live-mobile/src/analytics/segment.ts

```diff
@@ -98,6 +112,14 @@ const extraProperties = async (store: AppStore) => {
         ),
       ]
     : [];
+  const hasGenesisPass = hasNftInAccounts(
```

Comment:
Can you do a little benchmark of the impact  of this check ?

Using something like 

```
const t0 = performance.now();
doSomething();
const t1 = performance.now();
console.log(`Call to doSomething took ${t1 - t0} milliseconds.`);
```

On a portfolio having lot of NFTs without having the pass